donut/email_utils.py

```diff
@@ -2,7 +2,7 @@
 from email.mime.text import MIMEText
 
 
-def send_email(to, text, subject, use_prefix=True, group=None):
+def send_email(poster, to, text, subject, use_prefix=True, group=None):
```

Comment:
There are other places where we call `send_email()` in the code, e.g. for creating an account. Can you make this an optional parameter so that those pieces of code still work?